Dockerfile.deploy

```diff
@@ -1,10 +1,10 @@
-FROM adoptopenjdk/openjdk11:alpine-jre as builder
+FROM eclipse-temurin:17-jdk-alpine as builder
```

Comment:
Is it required to have this version or is it better to align everything to `amazoncorretto`?